frontend/public/components/monitoring/types.ts

```diff
@@ -50,19 +50,19 @@ export type MonitoringResource = {
 export type Silences = {
   data: Silence[];
   loaded: boolean;
-  loadError?: string;
+  loadError?: any;
```

Comment:
I thought `loadError` was always a `string`?

Reply:
Is passed into the action as an error directly from the request.
https://github.com/openshift/console/blob/master/frontend/public/actions/observe.ts#L78

Reply:
Could we make it `loadError?: string | Error`?